Code-Hammers/code-hammers
# [CHE 126] Add Remaining Profile Info To Edit Profile Form

### Description

As part of [CHE-113](https://codehammers.atlassian.net/browse/CHE-113) story "profiles improvement", this PR adds profile information to the edit profile form.

### Jira Task

[CHE-126](https://codehammers.atlassian.net/jira/software/projects/CHE/list?direction=ASC&selectedIssue=CHE-126&sortBy=labels)

### Testing Instructions

1. Make sure all branches are up to date and you are working from feature branch
2. Spin up the dev environment and log in 
3. Navigate to the edit profile page
4. Edit the information in each category
5. Hit the "Save Changes" button
6. Navigate back to the edit profile page and ensure all changes were saved to the profile

### Checklist

#### All Team Members


### Additional Notes, Images, etc.

N/A


[CHE-113]: https://codehammers.atlassian.net/browse/CHE-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-126]: https://codehammers.atlassian.net/browse/CHE-126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
